client/dist/index.html

```diff
@@ -2,12 +2,14 @@
 <html lang="en">
   <head>
     <meta charset="utf-8"/>
-    <title>SERUM</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="styles.css">
+    <title>SERUM</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,400;0,500;0,600;0,700;0,800;1,400;1,500;1,600;1,700;1,800&display=swap" rel="stylesheet">
```

Comment:
Same things, seems like to many things to load in

Reply:
Thanks! I wanted to load everything at first to experiment and see which families we actually need. I'll trim it down in a later PR once we've been able to make that determination.